examples/differential_xsec_plot.ma5

```diff
@@ -0,0 +1,25 @@
+# differential cross-section plot
+# c.f. http://madanalysis.irmp.ucl.ac.be/wiki/FAQNormalMode
+
+# install samples
+install samples
+
+# load sample and set cross-section
+import /root/madanalysis5/samples/zz.lhe.gz as my_sample
```

Comment:
This does have the disadvantage of being location specific, unless there is a way to import things starting from a relative path given by a shell variable.

Reply:
Unfortunately, this depends on where you execute `ma5` at the moment. The relative path is possible there is an anchor handle which can be set by
```
ma5> set main.currentdir = ...
```
this command is by default the execution location and everything can be added relative to this position. Once it is changed the relative paths must change as well. So for instance you executed from madanalysis5 folder
```
$ cd madanalysis5
$ ./bin/ma5
ma5> install samples
ma5> import samples/zz.lhe.gz as my_sample
```
should work since it attaches `currentdir` anchor at the beginning of the path string by default.